nixos/modules/services/web-apps/zabbix.nix

```diff
@@ -36,6 +36,10 @@ let
 
 in
 {
+  imports = [
+  (mkRenamedOptionModule
+      [ "services" "zabbixWeb" "virtualHost" ] [ "services" "zabbixWeb" "HttpdVirtualHost" ])
```

Comment:
let's do what other modules do... not this

Reply:
Could you elaborate on this?
I figured this is a good way to move preixisting installations to the new option. Previously it only used apache. With this PR nginx is also an option and with this `mkRenamedOptionModule` preexisting installations will continue to work as before

Reply:
sorry, that was really vague :sweat_smile:

`services.zabbixWeb.virtualHost` -> `services.zabbixWeb.httpd.virtualHost`

you should put all your `nginx` and `httpd` related options under `services.zabbixWeb.nginx` and `services.zabbixWeb`